ixmp4/data/db/optimization/indexset/repository.py

```diff
@@ -61,21 +61,36 @@ def list(self, *args, **kwargs) -> list[IndexSet]:
 
     @guard("view")
     def tabulate(self, *args, **kwargs) -> pd.DataFrame:
-        return super().tabulate(*args, **kwargs)
+        result = super().tabulate(*args, **kwargs).drop(labels="data_type", axis=1)
+        result.insert(
+            loc=0,
+            column="data",
+            value=[self.get_by_id(id=indexset_id).data for indexset_id in result.id],
+        )
+        return result
 
```

Comment:
This looks quite slow

Reply:
Agreed, but how else would I do this? Unfortunately, sqlalchemy doesn't consider these hybrid properties to be data columns, so our `tabulate()` function, which just selects the whole tables, only returns the new `data_type` column and not the `data` attribute. Though also not the `_data` relationship, I think.
I'll double check if this can be configured in a more automatic way.

Reply:
Would it be faster to adapt the `base` `tabulate()` function so it selects all indexsets while joining with `IndexSetData`?

Reply:
My first try would be to attempt to build sqlalchemy models form the data returned by tabulate directly or something like that.

Reply:
You recommended a flag to optionally load the `.data` from the relationship before, which I implemented now.

Reply:
Right, that's the better approach sorry